apps/backoffice-v2/src/common/hooks/useSearchParamsByEntity/validation-schemas.ts

```diff
@@ -3,11 +3,10 @@ import { CaseStatus, CaseStatuses, States } from '../../enums';
 
 export const SearchSchema = z.object({
   sortDir: z.enum(['asc', 'desc']).catch('desc'),
-  pageSize: z.coerce.number().int().positive().catch(10),
+  limit: z.coerce.number().int().positive().catch(50),
```

Comment:
Increased the page size because the search is in the client only, so at least it searches a big amount of cases.